README.md

```diff
@@ -35,9 +40,13 @@ This repository uses [lerna](https://github.com/lerna/lerna) to manage it's pack
 | react-shortcuts | [README](packages/react-shortcuts/README.md) | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-shortcuts.svg)](https://badge.fury.io/js/%40shopify%2Freact-shortcuts) |
 | with-env | [README](packages/with-env/README.md) | [![npm version](https://badge.fury.io/js/%40shopify%2Fwith-env.svg)](https://badge.fury.io/js/%40shopify%2Fwith-env) |
 
-## Contribute
+## Want to contribute?
+
+Check out our [Contributing Guide](./.github/CONTRIBUTING.md)
+
+## Questions?
 
-Check out our [Contributing Guidelines](CONTRIBUTING.md)
+For Shopifolk, you can reach out to us in Slack on the `#web-foundation` channel. For external inquiries, we welcome bug reports, enhancements, and feature requests via Github issues.
```

Comment:
We should still have the links to the guidelines here

Reply:
We do, just above in the Contributing section. This section is for other questions.